server/http/websocket/subscribe.go

```diff
@@ -62,6 +62,11 @@ func (e subscribeCommandExecutor) ResourceUpdatedEvent(resource interface{}) Eve
 		mapped = e.mappers.Out.Run(&v)
 	case *model.Run:
 		mapped = e.mappers.Out.Run(v)
+	case model.TransactionRun:
+		mapped = e.mappers.Out.TransactionRun(v)
+	case *model.TransactionRun:
+		var value model.TransactionRun = *v
+		mapped = e.mappers.Out.TransactionRun(value)
```

Comment:
Can you use this instead?

```suggestion
		mapped = e.mappers.Out.TransactionRun( *v)
```

Reply:
I tried that but the TransactionRun function expects a model instance value not a pointer 😢

Reply:
Are you sure you tried the exact code as Matheus suggested? It’s the exact same way we’ve done it for test runs, so it should work